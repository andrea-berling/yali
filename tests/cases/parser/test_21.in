(28 * -77 / (21 * 94))
