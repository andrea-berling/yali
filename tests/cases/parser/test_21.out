(group (/ (* 28.0 (- 77.0)) (group (* 21.0 94.0))))
